proto/minder/v1/minder.proto

```diff
@@ -1421,7 +1421,13 @@ message ListProfilesRequest {
     // The default is to return all user-created profiles; the string "*" can
     // be used to select all profiles, including system profiles.  This syntax
     // may be expanded in the future.
-    string label_filter = 2;
+    string label_filter = 2 [
+        (buf.validate.field).string = {
+            pattern: "^(\\*|[a-zA-Z][a-zA-Z0-9_]*)$",
```

Comment:
I think I was unable to find a usage of this server-side (though maybe that comment was on another PR?), so I'm not sure if it's in use at the moment.  I'd enforce lower-case on this for now.

Reply:
Addressed it 👍